airbyte-integrations/connectors/destination-s3-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/s3_v2/S3V2WriteTest.kt

```diff
@@ -40,6 +40,7 @@ abstract class S3V2WriteTest(
         commitDataIncrementally = commitDataIncrementally,
         allTypesBehavior = allTypesBehavior,
         nullEqualsUnset = nullEqualsUnset,
+        supportFileTransfer = false,
```

Comment:
```suggestion
        supportFileTransfer = true,
```

(I think? presumably the file transfer test was passing on s3)